src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> fruits = new HashMap<>();
+
+    public void addFruit(String fruit, int quantity) {
+        fruits.put(fruit, fruits.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public void removeFruit(String fruit, int quantity) {
+        fruits.put(fruit,fruits.getOrDefault(fruit, 0) - quantity);

```

Comment:
There should not be an empty line at the start of the `removeFruit` method implementation. Also, make sure there is a space after the comma in the `put` method call to follow Java code style conventions.